README.md

```diff
@@ -35,4 +35,14 @@ But usually, this back-office is useless unless you use its counterpart front-of
 ## Options
 
 TODO : Replace your notion API and datatlas port in the .env file.
-TODO : How to restrict access to the backoffice.
\ No newline at end of file
+TODO : How to restrict access to the backoffice.
+
+## Development
+
+For development installs all the node dependencies with
+`npm install`
+
+
+### Unit testing
+Unit testing is using Mocha
+`npm install -g mocha`
```

Comment:
Tests shouldn't depends on a global lib, `mocha` should be part of the `devDependencies`.

Reply:
Updated the README https://github.com/datatlas-erasme/back/pull/7